helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -141,11 +141,11 @@ internal class ListLicensesCommand : CliktCommand(
         val ortResult = ortResultFile.readValue<OrtResult>().replaceConfig(repositoryConfigurationFile)
 
         if (ortResult.getPackageOrProject(packageId) == null) {
-            throw UsageError("Could not find a package for the given id `$packageId`.")
+            throw UsageError("Could not find the package for the given id `${packageId.toCoordinates()}`.")
```

Comment:
Mind fixing the backticks to be single quotes in the same run?

Reply:
Done.